opensourceleg/hardware/actuators.py

```diff
@@ -497,6 +501,20 @@ def update(self) -> None:
                 motor_current=self.motor_current,
             )
 
+            if self.case_temperature > self.max_case_temperature:
+                self._log.warning(
+                    msg=f"[{str.upper(self._name)}] Case thermal limit {self.max_case_temperature} reached. Stopping motor."
+                )
+                self.stop()
+                exit()
+
+            if self.winding_temperature > self.max_winding_temperature:
```

Comment:
Am I missing where `self.winding_temperature` is unpacked from the thermal model?

Reply:
The `winding_temperature` is unpacked here: https://github.com/neurobionics/opensourceleg/blob/b74c342cec77111ef06715b8355984706f1dd256/opensourceleg/hardware/actuators.py#L877-L885